src/main/js/util/configurationRequestBuilder.js

```diff
@@ -71,6 +71,19 @@ export function createUpdateRequest(apiUrl, csrfToken, configurationId, fieldMod
     });
 }
 
+export function createUpdateWithoutIdRequest(apiUrl, csrfToken, model) {
```

Comment:
Do we validate that the role name is unique on the back-end? If not, we should.

Reply:
We do :)